### DISCUSSION TITLE
What is the exact data size limit for zincsearch?

### DISCUSSION THREAD

**Discussion by KamenRiderKuuga** [zinclabs/zinc#905] [Q&A] [answered]
We are currently planning to move from `Elasticsearch + Kibana` to `ZincSearch`. According to the known document content, we have learned that there is no high availability support for the time being, but I don't think our logs need such a thing.

What makes me more concerned is that I read in the README that  `Max data supported`  of  `ZincSearch` is `100s of GBs`. I wonder what this means and whether it will limit the storage of our logs.

**Comment by hengfeiyang** [comment, marked as answer]
First, it is no limits, but ZincSearch has no cluster mode, single node can't support a very huge data scale.
Second, If you use for Log system, I suggest you look at OpenObserve, this is another system we develop for logs,traces,metrics.

**Reply by KamenRiderKuuga** [reply]
Thanks for advice! We'd love to try the `OpenObserve`

In other words, just like the `100s of GBs` mentioned in the document, if our log exceeds 100 GB, maybe `ZincSearch` cannot support it?

**Reply by hengfeiyang** [reply]
100s GB means many 100GB

**Reply by KamenRiderKuuga** [reply]
I see

**Reply by lmlong** [reply]
Does "100s of GBs" refer to the size of the index data itself?
How much memory does it consume when dealing with 1 Million documents?

**Reply by hengfeiyang** [reply]
It based the document, maybe you document 1k or 1mb.
